hdx_hapi/endpoints/models/error.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Dict, List
+from pydantic import BaseModel
+from fastapi import status
+
+class BadRequestResponse(BaseModel):
+    detail: str
+
+class ResponseValidationResponse(BaseModel):
+    error_sample_list: List
+    error: str
+
+
+ERROR_RESPONSES: Dict[int, Dict[str, Any]] = {
+    status.HTTP_400_BAD_REQUEST: {
+        'model': BadRequestResponse,
```

Comment:
Potentially we need to handle 429 for the Datamart code since HDX should return a 429 status code on exceeding rate limit.